ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/customercenter/views/ManageSubscriptionsView.kt

```diff
@@ -129,106 +127,97 @@ private fun ActiveUserManagementView(
 @Composable
 private fun NoActiveUserManagementView(
     screen: CustomerCenterConfigData.Screen,
+    localization: CustomerCenterConfigData.Localization,
+    appearance: CustomerCenterConfigData.Appearance,
     onButtonPress: (CustomerCenterConfigData.HelpPath) -> Unit,
 ) {
-    Column {
+    Column(horizontalAlignment = Alignment.CenterHorizontally) {
         CompatibilityContentUnavailableView(
             title = screen.title,
-            drawableResId = R.drawable.warning,
             description = screen.subtitle,
+            modifier = Modifier.padding(PADDING_LARGE),
         )
 
         ManageSubscriptionsButtonsView(
-            screen = screen,
-            onButtonPress = onButtonPress,
-            useOutlinedButton = true,
+            screen,
+            localization,
+            appearance,
+            showSectionHeader = false,
+            onButtonPress,
         )
     }
 }
 
 @Composable
 fun CompatibilityContentUnavailableView(
     title: String,
-    drawableResId: Int,
     description: String?,
     modifier: Modifier = Modifier,
 ) {
-    Column(
-        modifier = modifier
-            .fillMaxWidth()
-            .padding(16.dp),
-        horizontalAlignment = Alignment.CenterHorizontally,
+    Surface(
+        modifier = modifier,
+        shape = MaterialTheme.shapes.medium,
+        color = MaterialTheme.colorScheme.surface,
     ) {
-        Image(
-            painter = painterResource(id = drawableResId),
-            contentDescription = null,
-            colorFilter = ColorFilter.tint(MaterialTheme.colorScheme.onBackground),
+        Column(
             modifier = Modifier
-                .size(48.dp)
-                .padding(bottom = 8.dp),
-        )
-
-        Text(
-            text = title,
-            style = MaterialTheme.typography.headlineSmall,
-            fontWeight = FontWeight.Bold,
-            modifier = Modifier.padding(bottom = 8.dp),
-        )
+                .fillMaxWidth()
+                .padding(PADDING_LARGE),
+            horizontalAlignment = Alignment.CenterHorizontally,
+        ) {
+            Icon(
+                imageVector = Icons.Rounded.Warning,
+                contentDescription = null,
+                tint = MaterialTheme.colorScheme.onSurface,
+                modifier = Modifier.size(ICON_SIZE),
+            )
 
-        description?.let {
             Text(
-                text = it,
-                style = MaterialTheme.typography.bodyLarge,
+                text = title,
+                style = MaterialTheme.typography.headlineSmall.copy(fontWeight = FontWeight.Bold),
                 modifier = Modifier.padding(bottom = 8.dp),
             )
+
+            description?.let {
+                Text(
+                    text = it,
+                    style = MaterialTheme.typography.bodyLarge,
+                    textAlign = TextAlign.Center,
+                    modifier = Modifier.padding(vertical = PADDING_LARGE),
```

Comment:
One thing I was noticing but if we have a description here, we would have a bottom padding of `PADDING_LARGE` which is 16.dp, but if not, the title bottom padding would be 8.dp instead. For consistency, maybe we should apply the overall paddings to the column, and just apply a top padding to the description if it exists?